tests/vmi_configuration_test.go

```diff
@@ -1113,7 +1113,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			},
 				table.Entry("[Serial][test_id:1671]hugepages-2Mi", "2Mi", "64Mi", "None"),
 				table.Entry("[Serial][test_id:1672]hugepages-1Gi", "1Gi", "1Gi", "None"),
-				table.Entry("[Serial][test_id:1672]hugepages-1Gi", "2Mi", "70Mi", "64Mi"),
+				table.Entry("[Serial][test_id:1672]hugepages-2Mi-guest-memory", "2Mi", "70Mi", "64Mi"),
```

Comment:
can we make the `-guest-memory` prefix a little bit more descriptive? Something like with guest memory explicitly set?

Reply:
Done